Gordon360/Models/ViewModels/RequestUploadViewModel.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Gordon360.Services;
+using System;
+
+namespace Gordon360.Models.CCT
+{
+    public partial class RequestUploadViewModel
+    {
+        public string ACTCode { get; set; }
+        public string SessCode { get; set; }
+        public string Username { get; set; }
+        public string PartCode { get; set; }
+        public DateTime DateSent { get; set; }
+        public string CommentText { get; set; }
+        public string Status { get; set; }
+
+        public REQUEST ToREQUEST()
+        {
+            return new REQUEST
+            {
+                ACT_CDE = this.ACTCode,
+                SESS_CDE = this.SessCode,
+                PART_CDE = this.PartCode,
+                DATE_SENT = this.DateSent,
+                COMMENT_TXT = this.CommentText,
+                STATUS = this.Status
+            };
+        }
+
+        public MembershipUploadViewModel ToMembershipUpload()
+        {
+            return new MembershipUploadViewModel
+            {
+                ACTCode = this.ACTCode,
+                SessCode = this.SessCode,
+                Username = this.Username,
+                PartCode = this.PartCode,
+                CommentText = this.CommentText,
+                GroupAdmin = false,
+                Privacy = false
+            };
+        }
```

Comment:
@EjPlatzer how do you feel about these methods being in the ViewModel? I like the idea of keeping them connected to the objects they are relevant to (at least one of them) rather than being in the service.

Reply:
I agree that it's nicer to have these attached to types rather than services. However, I think that it's cleaner (and more idiomatic C#) to use implicit operators here, rather than instance methods.